### DISCUSSION TITLE
Status Update

### DISCUSSION THREAD

**Discussion by rodit** [rodit/SnapMod#247] [Announcements]
Feature status moved to wiki [here](https://github.com/rodit/SnapMod/wiki/Features)

**Comment by rodit** [comment]
Hopefully I can fix these last few things and release an update soon

**Reply by RevealedSoulEven** [reply]
Too much eager for the updates sir.

**Comment by authorisation** [comment]
Could you maybe create a branch until everything is ready for release so volunteers can contribute (or at least try to)?

**Reply by RevealedSoulEven** [reply]
Ya bro. Why not? If it's done we can create something interesting too as we did in the past

**Reply by rodit** [reply]
I would but I will probably have everything ready tomorrow so I don't see much point - you are welcome to open a pr - most of the code in the repo is still relevant.

**Reply by RevealedSoulEven** [reply]
Just waiting for the update sir

**Comment by RevealedSoulEven** [comment]
@rodit why not adding a feature to Don't auto save chat messages for particular users as I created in my forked snapmod.

![Screenshot_2022-09-20-22-13-29-350_com snapchat android](https://user-images.githubusercontent.com/49313374/191316410-c8445289-99cc-45fc-b271-9dd28ec00474.jpg)
As you can see in the image I've added an option for don't auto save messages(see at last)

For example I've 200 friends and I want to save all chat messages except for one. So there is no way of doing it, so I created a way for exception. You can check in my repositories.

This is for the users who you don't want to auto save chat messages from a particular chat.
Please consider adding it to snapmod if you find it useful.


Thank you!

**Reply by rodit** [reply]
I will consider implementing a black vs whitelist mode for each of the toggles so you will be able to do this easily.

**Reply by RevealedSoulEven** [reply]
Yeaa sir. Sure🥺

**Reply by RevealedSoulEven** [reply]
Also please check #243 once

**Comment by RevealedSoulEven** [comment]
How longer do we need to wait for the update sir? Can't wait anymore for the comeback

**Reply by authorisation** [reply]
Chill out lol

**Reply by RevealedSoulEven** [reply]
Bro.... At every hour I check here to see whether a new update came or not. But still🥺💔

**Reply by GoatBan** [reply]
Different Time zone lol, we are not in hurry, the mod is still woking and your version is pretty good too

**Reply by RevealedSoulEven** [reply]
🔥thanks for the compliment brother

**Comment by RevealedSoulEven** [comment]
And I'm still refreshing the page every hour to check whether the update came or not🥺 Too much excited to see your comeback

**Reply by bih4st** [reply]
🥹

**Comment by authorisation** [comment]
It's out 🎉

**Reply by RevealedSoulEven** [reply]
Finally!!!! Something

**Comment by RevealedSoulEven** [comment]
It's been too long for the update @rodit

**Reply by Lucky-069** [reply]
Hey, my friend, constantly reminding him of this is becoming tiresome. We understand that you are looking forward to the bug fixes and update. But you forgot that this is a side project that he does not get paid for, and I'm sure there are many different changes that he has to make after the new updates, which will take some time. A late stable update is preferable to an early beta update.

**Comment by RevealedSoulEven** [comment]
For now I've fixed some issues in the previous version in my forked version. If anyone want to use it every feature in working great there. Check my profile.

**Comment by RevealedSoulEven** [comment]
No more new updates?

**Reply by GoatBan** [reply]
He's a student so he prolly dont have a lot of free time rn. Beside the mod is still working fine
